lib/Cron/Workers/SimpleWorker.php

```diff
@@ -97,12 +101,34 @@ function processTask(ScheduledTask $task) {
     // abort if execution limit is reached
     CronHelper::enforceExecutionLimit($this->timer);
 
-    if ($this->processTaskStrategy($task)) {
+    if (!static::SUPPORT_MULTIPLE_INSTANCES) {
+      if ($this->isInProgress($task)) {
```

Comment:
I think these two conditions (isInProgress and rescheduleOutdated) have to switch order. I had it wrong in my previous comment, sry :(
With the current order, it would not reschedule a task which is in progress.

Reply:
Good catch, this is a bug indeed, fixed.